shared/settings/index.js

```diff
@@ -10,7 +12,11 @@ class Settings extends Component {
   static parseRoute () {
     return {
       componentAtTop: {title: 'Settings'},
-      subRoutes: {devMenu},
+      subRoutes: {
+        devMenu,
+        deleteConfirm: DeleteConfirm,
+        removeDevice: RemoveDevice,
```

Comment:
mmm, could see it either way. Coyne doesn't have strong feelings either. On one hand I was to try and avoid having screens being arbitrarily pushed onto new paths, and switching tabs to device could actually be informational, but i dunno. Ultimately up to you.